tests/integration/api/test_serving_parent_features.py

```diff
@@ -157,6 +157,7 @@ def feature_list_deployment_with_child_entities_fixture(country_feature, mock_ta
     try:
         deployment = feature_list.deploy(make_production_ready=True)
         deployment.enable()
+        time.sleep(1)  # sleep 1s to invalidate cache
```

Comment:
if we do this more, we could wrap this in a utils function like

```
def sleep_to_invalidate_cache():
  time.sleep(1)
```

the benefit of this is that it'll be slightly easier to find the places where do this in the future (by looking for where the method is called) if we need to refactor/clean this up.